packages/backend/API.md

```diff
@@ -89,6 +90,9 @@ export { GenerateContainerEntryProps }
 
 export { ImportPathVerifier }
 
+// @public
+export type MainStackProps = Pick<StackProps, 'description' | 'env' | 'tags' | 'analyticsReporting' | 'crossRegionReferences' | 'suppressTemplateIndentation'>;

```

Comment:
Is this minimal set of picked properties to solve the problem? If not, please narrow it.

Reply:
Wasn't minimal for my specific issue. I have narrowed it in [e931b42](https://github.com/aws-amplify/amplify-backend/pull/1675/commits/e931b42cfb8eebc6a4023feff4fb41abd0e50560)